src/runner.py

```diff
@@ -87,9 +87,10 @@ def _schedule_test(self, tarball_node, plan, device):
 class RunnerLoop(Runner):
     """Runner subclass to execute in a loop"""
 
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, configs, args, **kwargs):
+        super().__init__(configs, args, **kwargs)
         self._job_tmp_dirs = {}
+        self._plan = self._plan_configs[args.plan]
```

Comment:
Getting error on this:

```
kernelci-pipeline-runner |     self._plan = self._plan_configs[args.plan]
kernelci-pipeline-runner | KeyError: 'check-describe'
```

Need to change below line in `kernelci.conf`:
```plan: check-describe```

Reply:
After I added `plan: kver` receiving errors:

```
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] Tarball: http://172.17.0.1:8002/linux-mainline-master-v5.19-rc8-105-g6e2c0490769e.tar.gz
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] Creating test node
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] Generating job
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] tmp: /home/kernelci/output/tmpxwsfewf8
kernelci-pipeline-notifier | 07/29/2022 06:30:56 AM UTC [INFO] 2022-07-29 06:30:56.859930  6e2c0490769e  Pending   NA       kver
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] output_file: /home/kernelci/output/tmpxwsfewf8/kver
kernelci-pipeline-runner | 07/29/2022 06:30:56 AM UTC [INFO] Running test
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | import-im6.q16: unable to open X server `' @ error/import.c/ImportImageCommand/346.
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 13: DB_CONFIG_YAML: not found
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 18: NODE_ID: not found
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 19: TARBALL_URL: not found
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 20: WORKSPACE: not found
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 21: REVISION: not found
kernelci-pipeline-runner | /home/kernelci/output/tmpxwsfewf8/kver: 25: Syntax error: "(" unexpected

```

Reply:
Yes that's fixed by other PRs. The `runtime` variable needs to be passed to the template, and I've dropped the default test plan names in `kernelci.conf` as it's now defined in `docker-compose.yaml`.

Reply:
I think the fixes for this issue should be included in this PR itself. Then we should merge this.
Or else the PR with fixes should be mentioned in the description as a dependency and the former should be merged first.

Reply:
Yes, there are just too many dependencies that I didn't list them all here.

Reply:
Actually I don't think this particular PR depends on any other changes, the `runtime` template parameter is already provided in the `main` branch. So I think the issue might be that you're testing with an older version of the `kernelci` package.